src/routes/Contact.tsx

```diff
@@ -0,0 +1,81 @@
+import styled from '@emotion/styled'
+import backgroundLogo from '../components/assets/bg_logo.svg'
+
+export default function Contact() {
+  return (
+    <$FormContact name="contact" method="POST" data-netlify="true">
+      <p>
+        <$Labels>
+          Name: <input type="text" name="name" />
+        </$Labels>
+      </p>
+      <p>
+        <$Labels>
+          Email: <input type="email" name="email" />
+        </$Labels>
+      </p>
+      <p>
+        <$Labels>
+          Role:
+          <select name="role">
+            <option value="">--Please choose an option--</option>
+            <option value="Maintainer">Maintainer</option>
+            <option value="Contributor">Contributor</option>
+            <option value="Guild Member">Guild Member</option>
+            <option value="Other">Other</option>
+          </select>
+        </$Labels>
+      </p>
+      <p>
+        <$Labels>
+          Message: <textarea name="message" />
+        </$Labels>
+      </p>
+      <p>
+        <$SubmitButton
+          type="submit"
+          onSubmit={(e) => {
```

Comment:
Nit: I enjoy typing the full `event`, just makes it more readable IMO. I leave it up to you.

Reply:
Sure. 👍🏻